pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -675,3 +678,53 @@ type DigitalOceanSDConfig struct {
 	// +optional
 	RefreshInterval *v1.Duration `json:"refreshInterval,omitempty"`
 }
+
+// KumaSDConfig allow retrieving scrape targets from Kuma's control plane.
+// See https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kuma_sd_config
+// +k8s:openapi-gen=true
+type KumaSDConfig struct {
+	// Address of the Kuma Control Plane's MADS xDS server.
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	Server *string `json:"server"`
```

Comment:
```suggestion
	Server string `json:"server"`
```

Let's not use pointers for required fields.

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required

Reply:
Thanks. Got it. change `*string` to `string`